docs/source/features/moe.rst

```diff
@@ -4,7 +4,7 @@ Mixture of Experts
 Overview
```

Comment:
Delete this heading here. We will move it to another location.

Overview
----------